op-challenger/game/fault/contracts/faultdisputegame2.go

```diff
@@ -0,0 +1,65 @@
+package contracts
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum-optimism/optimism/op-challenger/game/fault/types"
+	"github.com/ethereum-optimism/optimism/op-e2e/bindings"
+	"github.com/ethereum-optimism/optimism/op-service/sources/batching"
+	"github.com/ethereum-optimism/optimism/op-service/sources/batching/rpcblock"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+func (f *FaultDisputeGameContractLatest) GetSubClaims(ctx context.Context, block rpcblock.Block, aggClaim *types.Claim) ([]common.Hash, error) {
```

Comment:
The `block` parameter is never used, it may cause missing events.

Reply:
yeah, `block` param should be used. Changed `result, err := f.multiCaller.SingleCall(ctx, rpcblock.Latest, txCall)`  to	
`result, err := f.multiCaller.SingleCall(ctx, block, txCall)`

Done in https://github.com/ethstorage/optimism/pull/63/commits/16c4c67fbf5aa6e98e8d674fdfcb37294b0dd0cc

Reply:
Is it needed to pass the block info to `FilterMove`'s 1st param?

Reply:
Thanks. Done in https://github.com/ethstorage/optimism/pull/63/commits/0fbfd260ebfd7cbb03f110ccf8a7b9a5c2f8425b